Wikipedia/Code/WMFAppViewController.m

```diff
@@ -1576,22 +1578,31 @@ - (void)presentOnboardingIfNeededWithCompletion:(void (^)(BOOL didShowOnboarding
 #pragma mark - Splash
 
 - (void)showSplashView {
+    if (_splashScreenViewController) {
```

Comment:
Would you be opposed to using self notation `self.splashScreenViewController` instead of `_splashScreenViewController`? It would be a little more consistent with how the other properties (`self.periodicWorkerController`, etc.) are called in here, and I think it will still function the same. I generally try to avoid accessing the instance variable directly outside of getters and setters for Obj-C.

Reply:
I agree! Code updated, thank you

Reply:
@tonisevener I'm a stranger to this process, should the phabricator ticket be moved back to "Needs code review" or should it be left as is? 🙏

Reply:
@christianbilodeau You can move it back into "Needs code review". Thanks!